frontend/src/components/ItemTable.tsx

```diff
@@ -65,8 +65,8 @@ function ItemTable(props: itemPageProbs) {
         if (itemToAdd === undefined) {
             return onCancel();
         }
-        if (typeof valueStoreable === "string") {
-            itemToAdd.storeableValue = valueStoreable;
+        if (typeof valueStorable === "string") {
+            itemToAdd.storableValue = valueStorable;
```

Comment:
Isn't this condition always true since the useState is defined as being a string?
Maybe you rather wanted to check if it is null/empty/undefined?